src/sparc/client/client.py

```diff
@@ -69,39 +67,52 @@ def __init__(self, config_file="config/config.ini", connect=True) -> None:
                 f"{__package__}.services.{module_name}", config[current_config], connect
             )
 
-    def add_module(self, path, config, connect=True):
+    #        else:
+    #            self.add_module(package_dir, config[current_config], connect)
```

Comment:
Shouldn't we avoid having commented code in the main branch? It does not bring anything and just adds noise in a shared codebase

Only in exceptional cases but the reason should be well documented (e.g. ``# NOTE: this code is left here because ... `` )

Reply:
Good catch!